modules/rosa-hcp-custom-create-network.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-hcp-custom-create-network_{context}"]
+= Configuring and using custom CloudFormation templates
```

Comment:
This does seem to be only "Creating a network using a custom CloudFormation template" - which is in theory something you can do by passing a parameter in the normal process, right?

Would it make more sense to remove this module and direct users to AWS CloudFormation docs if they need to create their own template?